qcodes/instrument_drivers/Keysight/keysightb1500/KeysightB1520A.py

```diff
@@ -63,7 +63,7 @@ def __init__(self, parent: 'B1520A', name: str, **kwargs: Any):
         self.post_sweep_voltage_condition.cache.set(constants.WMDCV.Post.START)
 
         self.add_parameter(name='hold',
-                           initial_value=0,
+                           initial_value=0.0,
```

Comment:
The tests expect these values to be 0.0 initially so we change them to this

Reply:
ok